src/main/java/com/gtnewhorizons/angelica/config/AngelicaConfig.java

```diff
@@ -110,6 +110,11 @@ public class AngelicaConfig {
     @Config.RequiresMcRestart
     public static boolean fixExtraUtilsSodiumCompat;
 
+    @Config.Comment("Fix RenderBlockFluid reading the block type from the world access multiple times")
```

Comment:
I'd probably just globally apply this with Angelica, but I suppose a separate config doesn't hurt.

Reply:
It might be useful for further debugging if issues arise with this code